tests/unit/dataSource/properties/tests.html

```diff
@@ -1019,15 +1019,17 @@
 				"Verify that all parameters were properly encoded.");
 	 		
 	 		dsFilter.settings.filtering.filterExprUrlKey = null;
-			var prevMonth = date.getMonth() % 12;
+			var prevMonthDate = new Date(new Date(ticks).setMonth(-1));
+			var prevMonth =prevMonthDate.getMonth() + 1;
 	 		var currMonth = date.getMonth() % 12 + 1;
```

Comment:
`var currMonth = date.getMonth() % 12 + 1; `

You don't need the `% 12` for this one.